### ISSUE TITLE
Duplicate module name: promiseRejectionIsError

### ISSUE THREAD

**Issue by shiv86** [xcarpentier/react-native-stripe-api#1]
This error is caused by a @providesModule declaration with the same name accross two different files.
Error: @providesModule naming collision:
  Duplicate module name: promiseRejectionIsError
  Paths: 

> projectname/node_modules/react-native-stripe-api/node_modules/react-native/Libraries/promiseRejectionIsError.js collides with 

> projectname/node_modules/react-native/Libraries/promiseRejectionIsError.js

**Comment by shiv86** [comment]
Problem: This package seems to be installing another react and react-native module which conflict with the over all project modules.

The reason for this appears to be because a specific version of both these modules are defined as a devDependency in the package.json:

```
  "dependencies": {
    "babel-polyfill": "6.9.1",
    "react": "15.1.0",
    "react-native": "0.27.2"
  },
```

It seems like these should be removed.

Could someone with more knowledge about npm confirm this is actually the correct way of defining node dependencies. And the appropriate  solution to resolve the problem

**Comment by shiv86** [comment]
I am using:

Project version:
    "react": "^15.2.1",
    "react-native": "^0.30.0",

NPM:
  node: '5.0.0',
  npm: '3.3.6',

**Comment by shiv86** [comment]
Until pull request #1 is accept feel free to use this fork:

npm install --save https://github.com/shiv86/react-native-stripe-api.git

**Comment by xcarpentier** [comment]
Thanks @shiv86 !
Fixed on 0.0.5.
